examples/cd_md3_winequality.ipynb

```diff
@@ -0,0 +1,593 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Margin density drift detection on wine quality dataset\n",
```

Comment:
Was Wine Quality an existing dataset in the project already? Or is it new with this addition?

If it's whole new and it's not a huge inconvenience, we'd prefer to steer clear of Alcohol and Tobacco references.

If it's an existing dataset in the project, then it's ok to follow prior work.

Reply:
Thank you for your feedback!

No, the wine quality dataset is not an existing dataset in the project and the project did not have a suitable, existing dataset available.  Technically this dataset is being referenced by the project but not actually added to the project (i.e, stored in the project's cloud storage, fetched and preprocessed with custom methods that are abstracted away from the user, etc.).  The dataset is being downloaded and fetched from a different library called tensorflow-datasets.  Anyway, after searching I could not find a reasonable alternative.  The wine quality dataset was selected because it met the following requirements: referenced in original paper as a benchmark, readily downloadable from any open-source library (such as alibi, alibi-detect, tensorflow-datasets, scitkit-learn, etc.), has requisite number of instances to train a TF/Keras model, tabular format, binary target variable or easily converted to binary target, and numeric and normally-distributed features so that the induced drift would make sense.  For these reasons along with the fact that the 'wine quality' is a classic dataset that most ML practitioners already familiar with I'm seeking exemption from the requirement that there be no references to tobacco or alcohol in open-source contributions.

Reply:
> the 'wine quality' is a classic dataset that most ML practitioners

If it's a community norm, go for it. Approved.